test/controllers/api/solutions/mentor_discussion_posts_controller_test.rb

```diff
@@ -33,6 +33,7 @@ class API::Solutions::MentorDiscussionPostsControllerTest < API::BaseTestCase
           uuid: "request-comment",
           iteration_idx: 2,
           author_handle: "student",
+          author_flair: student&.flair,
```

Comment:
This also applies to the other occurences in this file

```suggestion
          author_flair: student.flair,
```